webservice/controller/admin_assignment.go

```diff
@@ -386,3 +386,97 @@ func AdminUpdateAssignmentPOST(w http.ResponseWriter, r *http.Request) {
 
 	http.Redirect(w, r, "/admin/assignment", http.StatusFound)
 }
+
+// AdminAssignmentSubmissionsGET servers list of all users in course to admin
+func AdminAssignmentSubmissionsGET(w http.ResponseWriter, r *http.Request) {
+
+	vars := mux.Vars(r)
+	assignmentID, err := strconv.Atoi(vars["id"])
+	if err != nil {
+		log.Printf("id: %v", err)
+		ErrorHandler(w, r, http.StatusInternalServerError)
+		return
+	}
+
+	w.Header().Set("Content-Type", "text/html; charset=utf-8")
+	w.WriteHeader(http.StatusOK)
```

Comment:
This could cause double writing header. Should be put after all: 
```go
if err != nil {
	log.Println(err.Error())
	ErrorHandler(w, r, http.StatusInternalServerError)
	return
}
```